cromulent/reader.py

```diff
@@ -1,116 +1,128 @@
+        what = clx(ident=ident)
+        self.uri_object_map[ident] = what
+        propInfo = what._list_all_props()
 
-		classified = js.get('classfied_as', [])
-		what = clx(ident=ident)
-		self.uri_object_map[ident] = what
-		propInfo = what._list_all_props()
+        # sort data by KOH to minimize chance of bad backrefs
+        itms = list(js.items())
+        itms.sort(key=lambda x: koh.get(x[0], 10000))
 
-		# sort data by KOH to minimize chance of bad backrefs
-		itms = list(js.items())
-		itms.sort(key=lambda x: koh.get(x[0], 10000))
+        for (prop, value) in itms:
+            # Make everything a list, even if it can't be one
+            rng = propInfo.get(prop, None)
+            if not rng:
+                raise DataError("Unknown property %s" % prop)
 
-		for (prop, value) in itms:
-			# Make everything a list, even if it can't be one
-			rng = propInfo.get(prop, None)
-			if not rng:
-				raise DataError("Unknown property %s" % prop)
-	 
-			if type(value) != list:
-				value = [value]
-			for subvalue in value:
-				if rng == str:
-					setattr(what, prop, subvalue)				
-				elif type(subvalue) == dict or isinstance(subvalue, OrderedDict):
-					# recurse ...
-					val = self.construct(subvalue)
-					setattr(what, prop, val)
-				elif type(subvalue) in STR_TYPES:
-					# raw URI to be made into a class of type rng
-					# or back reference
-					if subvalue in self.uri_object_map:
-						setattr(what, prop, self.uri_object_map[subvalue])
-					elif rng in [model.Type, BaseResource]:
-						# Always a X, often no more info
-						setattr(what, prop, rng(ident=subvalue))
-					else:
-						self.forward_refs.append([what, prop, subvalue])
-				else:
-					# No idea!!
-					raise DataError("Value %r is not expected for %s" % (subvalue, prop))
+            if type(value) != list:
+                value = [value]
+            for subvalue in value:
+                if rng == str:
+                    setattr(what, prop, subvalue)
+                elif type(subvalue) == dict or isinstance(subvalue, OrderedDict):
+                    # recurse ...
+                    val = self.construct(subvalue)
+                    setattr(what, prop, val)
+                elif type(subvalue) in STR_TYPES:
+                    # raw URI to be made into a class of type rng
+                    # or back reference
+                    if subvalue in self.uri_object_map:
+                        setattr(what, prop, self.uri_object_map[subvalue])
+                    elif rng in [model.Type, BaseResource]:
+                        # Always a X, often no more info
+                        setattr(what, prop, rng(ident=subvalue))
+                    else:
+                        self.forward_refs.append([what, prop, subvalue])
+                else:
+                    # No idea!!
+                    raise DataError(
+                        "Value %r is not expected for %s" % (subvalue, prop))
 
-		return what
+        return what
```

Comment:
This all looks like unnecessary whitespace changes that could be fixed with an IDE setting.